Remove ENABLE(CSS_SELECTORS_LEVEL4) build flag

rniwa opened the pull request:
#### 609b24aff7be93e4d6349b2528b21bce30c17c0e
<pre>
Remove ENABLE(CSS_SELECTORS_LEVEL4) build flag
<a href="https://bugs.webkit.org/show_bug.cgi?id=243196">https://bugs.webkit.org/show_bug.cgi?id=243196</a>

Reviewed by Tim Nguyen.

Removed the build flag since it&apos;s enabled everywhere now.

* Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml:
* Source/WTF/wtf/PlatformEnable.h:
* Source/WebCore/css/CSSSelector.cpp:
(WebCore::appendPseudoClassFunctionTail):
(WebCore::CSSSelector::selectorText const):
* Source/WebCore/css/CSSSelector.h:
* Source/WebCore/css/SelectorChecker.cpp:
(WebCore::SelectorChecker::checkOne const):
* Source/WebCore/css/SelectorCheckerTestFunctions.h:
(WebCore::matchesDirPseudoClass):
* Source/WebCore/css/SelectorPseudoClassAndCompatibilityElementMap.in:
* Source/WebCore/css/parser/CSSSelectorParser.cpp:
(WebCore::isOnlyPseudoClassFunction):
(WebCore::CSSSelectorParser::consumePseudo):
* Source/WebCore/cssjit/SelectorCompiler.cpp:
(WebCore::SelectorCompiler::addPseudoClassType):
* Source/cmake/OptionsWin.cmake:
* Source/cmake/WebKitFeatures.cmake:
* Source/cmake/tools/vsprops/FeatureDefines.props:
* Source/cmake/tools/vsprops/FeatureDefinesCairo.props:

Canonical link: <a href="https://commits.webkit.org/252840@main">https://commits.webkit.org/252840@main</a>
</pre>

nt1m reviewed (approved):
r=me if EWS is happy

rniwa committed:
Remove ENABLE(CSS_SELECTORS_LEVEL4) build flag
https://bugs.webkit.org/show_bug.cgi?id=243196

Reviewed by Tim Nguyen.

Removed the build flag since it's enabled everywhere now.

* Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml:
* Source/WTF/wtf/PlatformEnable.h:
* Source/WebCore/css/CSSSelector.cpp:
(WebCore::appendPseudoClassFunctionTail):
(WebCore::CSSSelector::selectorText const):
* Source/WebCore/css/CSSSelector.h:
* Source/WebCore/css/SelectorChecker.cpp:
(WebCore::SelectorChecker::checkOne const):
* Source/WebCore/css/SelectorCheckerTestFunctions.h:
(WebCore::matchesDirPseudoClass):
* Source/WebCore/css/SelectorPseudoClassAndCompatibilityElementMap.in:
* Source/WebCore/css/parser/CSSSelectorParser.cpp:
(WebCore::isOnlyPseudoClassFunction):
(WebCore::CSSSelectorParser::consumePseudo):
* Source/WebCore/cssjit/SelectorCompiler.cpp:
(WebCore::SelectorCompiler::addPseudoClassType):
* Source/cmake/OptionsWin.cmake:
* Source/cmake/WebKitFeatures.cmake:
* Source/cmake/tools/vsprops/FeatureDefines.props:
* Source/cmake/tools/vsprops/FeatureDefinesCairo.props:

Canonical link: https://commits.webkit.org/252840@main
